File: MyStack/src/main/java/Node.java
/**
 * This class is node for stack.
 * @param <E>
 */
public class Node<E> {
  private Node<E> next = null;
  private E value;

  public Node(E value, Node next) {
    this.value = value;
    this.next = next;
  }
  public Node getNext(){
    return next;
  }
  public void setNext(Node next) {
    this.next = next;
  }
  public void setData(E value) {
    this.value = value;
  }
  public E getData() {
    return value;
  }
}

File: StackProject/StackProject/Node.cs
namespace StackProject
{
  /// <summary>
  /// This is node for Stack.
  /// </summary>
  /// <typeparam name="T"> Any type of value </typeparam>
  public class Node<T>
  {
    public T Value { get; set; }
    public Node<T> Next { get; set; }

    /// <summary>
    /// This is constructor for Node class.
    /// </summary>
    /// <param name="value"> Any value </param>
    public Node(T value)
    {
      Value = value;
    }
  }
}